Makefile

```diff
@@ -1,5 +1,7 @@
 .DEFAULT_GOAL := help
 
+DOCKER_BUILDER_IMAGE_NAME = govuk/notify-net-client-builder
```

Comment:
Our images are now given slightly better names, you can see them in https://hub.docker.com/u/govuknotify

So this one should be `govuknotify/net-client-tests`